uc_mcore/vuln_tweetnacl.c

```diff
@@ -0,0 +1,811 @@
+#include "vuln_tweetnacl.h"
```

Comment:
Can we add some more info regarding the carry bug - either to the readme or even comment some lines here and mark where it happens/occurs?

We could also link to some description if there is any.